docs/book/src/getting-started.md

```diff
@@ -116,4 +116,11 @@ it into a management cluster using `clusterctl`.
    export CLUSTER_TOPOLOGY=true
    ```
       
+   Similarly, to deploy workload cluster with [VPC clusterclass-template](/topics/VPC/clusterclass-cluster.html). Set the following environmental variables.
+
+   ```console
+   export PROVIDER_ID_FORMAT=v2
```

Comment:
We need to pass  `EXP_CLUSTER_RESOURCE_SET` also right as mentioned in the doc below? If yes, would be good to combine the code snippets for PowerVS and VPC ClusterClass to avoid repetition.

Reply:
IIRC, the `PROVIDER_ID_FORMAT` environment variable was not required. However, the `EXP_CLUSTER_RESOURCE_SET` variable needs to be set to `true`. I have made the changes accordingly and have updated the documentation. This was a miss from my end.

Thank you.

Reply:
I have updated the docs as suggested, this was a miss from my end.
Thank you for pointing out..